src/executables/util_service.c

```diff
@@ -3950,50 +3950,13 @@ us_hb_process_start (HA_CONF * ha_conf, const char *db_name, bool check_result)
 
   print_message (stdout, MSGCAT_UTIL_GENERIC_START_STOP_2S, PRINT_HA_PROCS_NAME, PRINT_CMD_START);
 
-  pids = da_create (100, sizeof (int));
-  if (pids == NULL)
-    {
-      status = ER_GENERIC_ERROR;
-      goto ret;
-    }
-
   status = us_hb_server_start (ha_conf, db_name);
   if (status != NO_ERROR)
     {
       goto ret;
     }
 
-  status = us_hb_copylogdb_start (pids, ha_conf, db_name, NULL, NULL);
-  if (status != NO_ERROR)
-    {
-      goto ret;
-    }
-
-  status = us_hb_applylogdb_start (pids, ha_conf, db_name, NULL, NULL);
-  if (status != NO_ERROR)
-    {
-      goto ret;
-    }
-
-  sleep (HB_START_WAITING_TIME_IN_SECS);
```

Comment:
This part is that, 
Sleep for a while, and checks if copylogdb and applylogdb are well executed (check if alive)